src/trackers/TVC.py

```
@@ -251,97 +303,150 @@ async def upload(self, meta, disctype):
             data['episode_number'] = meta.get('episode_int', '0')
-        headers = {
-            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:53.0) Gecko/20100101 Firefox/53.0'
-        }
-        params = {
-            'api_token': self.config['TRACKERS'][self.tracker]['api_key'].strip()
-        }
+
         if 'upload_to_tvc' in locals() and upload_to_tvc is False:
             return
 
+        torrent_path = f"{meta['base_dir']}/tmp/{meta['uuid']}/[{self.tracker}].torrent"
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)